onnxruntime/core/graph/function_utils.cc

```diff
@@ -269,7 +269,7 @@ static void IOTypeConstraintHelper(const ONNX_NAMESPACE::FunctionProto& onnx_fun
 
 std::unique_ptr<ONNX_NAMESPACE::OpSchema> CreateSchema(const std::string& function_domain,
                                                        const std::string& function_name,
-                                                       const InlinedHashMap<std::string, const ONNX_NAMESPACE::FunctionProto*>& model_local_functions,
+                                                       const std::unordered_map<std::string, const ONNX_NAMESPACE::FunctionProto*>& model_local_functions,
```

Comment:
Would be good to add a note/comment as to why we're choosing to use unordered_map here even when the coding conventions ask us to use InlinedHashMap.

Reply:
We should update the coding conventions doc too to qualify the usage of these containers.